tests/python/common/data/test_data.py

```diff
@@ -1668,6 +1668,9 @@ def _test_NodeEdgeGraphData():
     reason="Datasets don't need to be tested on GPU.",
 )
 @unittest.skipIf(dgl.backend.backend_name == "mxnet", reason="Skip MXNet")
+@unittest.skipIf(
```

Comment:
due to `AttributeError: module 'typing' has no attribute 'OrderedDict'`. seems need to upgrade python version